src/chainparams.cpp

```diff
@@ -138,6 +140,23 @@ class CMainParams : public CChainParams {
                  //   (the tx=... number in the SetBestChain debug.log lines)
             0    // * estimated number of transactions per day after checkpoint
         };
+
+        // Founders reward script expects a vector of 2-of-3 multisig addresses
+        vFoundersRewardAddress = {
+            "ADDRESS1", "ADDRESS2", "ADDRESS3", "ADDRESS4",
+            "ADDRESS5", "ADDRESS6", "ADDRESS7", "ADDRESS8",
+            "ADDRESS9", "ADDRESS10", "ADDRESS11", "ADDRESS12",
+            "ADDRESS13", "ADDRESS14", "ADDRESS15", "ADDRESS16",
+            "ADDRESS17", "ADDRESS18", "ADDRESS19", "ADDRESS20",
+            "ADDRESS21", "ADDRESS22", "ADDRESS23", "ADDRESS24",
+            "ADDRESS25", "ADDRESS26", "ADDRESS27", "ADDRESS28",
+            "ADDRESS29", "ADDRESS30", "ADDRESS31", "ADDRESS32",
+            "ADDRESS33", "ADDRESS34", "ADDRESS35", "ADDRESS36",
+            "ADDRESS37", "ADDRESS38", "ADDRESS39", "ADDRESS40",
+            "ADDRESS41", "ADDRESS42", "ADDRESS43", "ADDRESS44",
+            "ADDRESS45", "ADDRESS46", "ADDRESS47", "ADDRESS48",
+            };
+        assert(vFoundersRewardAddress.size() < consensus.nSubsidyHalvingInterval + consensus.SubsidySlowStartShift());
```

Comment:
This should be rewritten in terms of `GetLastFoundersRewardBlockHeight()`

Non-blocking.